nd4j/nd4j-backends/nd4j-api-parent/nd4j-api/src/main/java/org/nd4j/linalg/api/ops/impl/layers/recurrent/SRUCell.java

```diff
@@ -46,6 +47,14 @@ public SRUCell(SameDiff sameDiff, SDVariable x, SDVariable cLast, SRUWeights wei
         this.weights = weights;
     }
 
+
+
+    public SRUCell(INDArray x, INDArray cLast, SRUWeights sruWeights) {
+        super(null, null, sruWeights.argsWithInputs(x, cLast));
+        this.weights = sruWeights;
+    }
```

Comment:
Bad constructor. You can't mix SDVariable and INDArray like this.
That superclass call is to `DynamicCustomOp(String opName, SameDiff sameDiff, SDVariable[] args)`
It should be to `DynamicCustomOp(INDArray[] inputs, INDArray[] outputs)`